src/nominatim_db/data/lang_info.py

```diff
@@ -0,0 +1,79 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
This file must be available in `nominatim_api` if it is to be used there. Move this file simply over there. For `country_info.py` create a smaller version that only contains the get functions you need.